NAS2D/Resource/Font.cpp

```diff
@@ -69,7 +69,7 @@ namespace
  * \param	ptSize		Point size of the font. Defaults to 12pt.
  */
 Font::Font(const std::string& filePath, unsigned int ptSize) :
-	mResourceName{filePath + "_" + std::to_string(ptSize) + "pt"},
+	mFilePath{filePath + "_" + std::to_string(ptSize) + "pt"},
 	mFontInfo{load(filePath, ptSize)}
```

Comment:
I believe it was called `mResourceName` because presumably not all resources would be loaded from files. In particular, take a look at the `Image` resource, which can be created from raw pixel data.

All resources use the same naming convention. That stems partly from them once having a shared based class with that field in it. Though it may make sense to keep the same naming convention between resource classes.

Honestly though, I'm tempted to get rid of the field, so maybe it doesn't matter. I guess it's somewhat useful for error messages, or debugging, but not much outside of that.

Reply:
It was originally for debugging purposes and I had this idea of being able to 'find' resources/assets from a 'resource manager' type of deal by name. That was .... wow that was a long time ago (approaching 20 years, damn). Anyway, it can be removed especially with the `FontCache` object introduced in OPHD (don't remember if that's built into NAS2D or an OPHD exclusive).